Assets/MRTK/Providers/Experimental/WindowsSceneUnderstanding/WindowsSceneUnderstandingObserver.cs

```diff
@@ -131,20 +146,23 @@ public override void Initialize()
             {
                 observerState = ObserverState.GetScene;
             }
+#endif // SCENE_UNDERSTANDING_PRESENT
         }
 
         /// <inheritdoc />
         public override void Enable()
         {
+#if SCENE_UNDERSTANDING_PRESENT
+            base.Enable();
```

Comment:
we may need an

```cs
#else
    IsEnabled = false;
#endif
```

to prevent logs like https://github.com/microsoft/MixedRealityToolkit-Unity/issues/9392 (`Microsoft.MixedReality.Toolkit.XRSDK.Oculus.Input.OculusXRSDKDeviceManager has not set a value for IsEnabled, returning false.`)